Frontend/src/app/components/product-details/product-details.component.spec.ts

```diff
@@ -72,4 +87,10 @@ describe('ProductDetailsComponent', () => {
     it('should create', () => {
         expect(component).toBeTruthy();
     });
+
+    it('should show the product name', () => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Mueller-Patrick/Betterzon/pullRequest?prid=7401082)
```suggestion
    it("should show the product name", () => {
```